model/task/results.go

```diff
@@ -133,3 +134,38 @@ func FilterTasksOnStatus(tasks []Task, statuses ...string) []Task {
 
 	return out
 }
+
+type Result struct {
+	Name  string `json:"name"`
+	Count int    `json:"count"`
+}
+type ResultCountList struct {
+	Total              []Result `json:"total"`
+	Inactive           []Result `json:"inactive"`
+	Unstarted          []Result `json:"unstarted"`
+	Started            []Result `json:"started"`
+	Succeeded          []Result `json:"succeeded"`
+	Failed             []Result `json:"failed"`
+	SetupFailed        []Result `json:"setup-failed"`
+	SystemFailed       []Result `json:"system-failed"`
+	SystemUnresponsive []Result `json:"system-unresponsive"`
+	SystemTimedOut     []Result `json:"system-timed-out"`
+	TestTimedOut       []Result `json:"test-timed-out"`
+}
```

Comment:
I'd replace this with a map[string]Result{}

Reply:
The objective I was trying to accomplish was to list them in ascending order.
When a map is mashalled into json the keys are ordered lexicographically, so it would be "1", "10", "2", "3"...

Reply:
descending order of what? This is static. I don't think that we should pay particular order here

Reply:
I misunderstood, but I get it now.
It's now a map[status]:[]{Name,Count} (Where the list of {Name, Count} is sorted in descending order by count).

This has the additional advantage that adding statuses should work without any changes.